style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  max-width: 1024px;
+  margin: 0 auto;
+  background-color: #EFEFEF;
+}
+
+h1 {
+  margin-bottom: 80px;
+  padding-top: 40px;
+  font-family: sans-serif;
+  font-size: 18px;
+}
+
+h2 {
+  margin-top: 40px;
+  padding-left: 50px;
+  font-size: 16px;
+}
+
+figure {
+  margin-right: 60px;
+  padding: 10px;
+  float: left;
+  text-align: center;
+  font-family: sans-serif;
+}
+
+.none-marker > li {
+  list-style-type: none;
+  text-indent: 20px;
+}
+
+li {
+  margin-left: 40px;
+  padding-bottom: 10px;
+  color: #222222;
+  list-style-type: square;
+  text-indent: 10px;
+}
+
+ul > ul > li {
+  list-style-type: square;
+}
+
+.contacts li > a {
+  text-decoration: none;
+  color: black;
+}
+
+.clear {
+  padding-top: 40px;
+  clear: both;
+}
+
+code {
+  float: right;

```

Comment:
`float: right;` is unnecessary.
Better to set `display: block` and add `margin-left.` In this case you will not need in `clear: both`.

BTW: tag `<pre>` is a block element.